winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+        WinRT.Interop.InitializeWithWindow.Initialize(savePicker, hwnd);
+        stFile = await savePicker.PickSaveFileAsync();
+    }
+    else
+    {
+        StorageFolder local = Windows.Storage.ApplicationData.Current.LocalFolder;
+        stFile = await local.CreateFileAsync(filename, CreationCollisionOption.ReplaceExisting);
+    }
+    if (stFile != null)
+    {
+        using (IRandomAccessStream zipStream = await stFile.OpenAsync(FileAccessMode.ReadWrite))
+        {
+            //Write the compressed data from the memory to the file
+            using (Stream outstream = zipStream.AsStreamForWrite())
+            {
+                byte[] buffer = stream.ToArray();
+                outstream.Write(buffer, 0, buffer.Length);
+                outstream.Flush();
+            }
+        }
+        //Launch the saved Excel file.
+        await Windows.System.Launcher.LaunchFileAsync(stFile);
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> The SfTreeGrid exports data to excel by using [XlsIO](http://help.syncfusion.com/file-formats/xlsio/overview). Refer to the [XlsIO documentation](http://help.syncfusion.com/file-formats/xlsio/working-with-excel-worksheet) for manipulating the exported worksheets.
+
+## Excel exporting options
+
+The exporting operation can be customized by passing `TreeGridExcelExportOptions` instance as an argument to the `ExportToExcel` method. 
+
+### Export mode
+
+By default, only the actual value will be exported to excel. To export the display text, set the `ExportMode` property as `Text`. 
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExportMode = ExportMode.Text;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Export without outlines	
+
+By default, all the children in the TreeGrid will be exported based on the state in TreeGrid. You can disable the outlines in Excel by setting the `ShowOutlines` property to false in `TreeGridExcelExportOptions` class.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.ShowOutlines = false;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-export-to-excel.png" alt="WinUI TreeGrid Data Exporting to Excel without Outlines" width="100%" Height="Auto"/>
+
+### Exclude columns while exporting
+
+By default, all the columns (including hidden columns) in the TreeGrid will be exported to Excel. To exclude some columns, use `ExcludedColumns` field in `TreeGridExcelExportOptions` class.

```

Comment:
By default, all the columns (including hidden columns) in TreeGrid will be exported to Excel. To exclude some columns, use the `ExcludedColumns` field in the `TreeGridExcelExportOptions` class.